### PR TITLE
Fixed testwebfiles

### PR DESCRIPTION
**Pull request by AlbaMontis**

### CODE REVIEW

File: tests/testwebfiles.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ function testWebFiles() {
   for (; length;) {
     text = elements[--length].textContent.trim();
     code = +text.split(': ')[1];
-    if (!code || !~HTTP_STATUS_CODES.indexOf(code)) {
+    if (!text || !~HTTP_STATUS_CODES.indexOf(code)) {
```
Comment (vpodk) [inline tests/testwebfiles.js]: Why you checking `text` instead of `code`?

**Comment by vpodk** [comment]
Also please add success criteria description.

File: tests/testwebfiles.js (hunk 1 above)
Comment (AlbaMontis) [inline tests/testwebfiles.js]: code can be  0,  !0 == true, so  test will fail.

Comment (vpodk) [inline tests/testwebfiles.js]: If text is an empty string?

``` javascript
!'' == true
```

Comment (vpodk) [inline tests/testwebfiles.js]:
``` javascript
isNaN(code)
```

**Pull request closed without merging**